sos/report/plugins/nvme.py

```diff
@@ -26,6 +26,7 @@ class Nvme(Plugin, IndependentPlugin):
     kernel_mods = ('nvme', 'nvme_core')
 
     def setup(self):
+        _subdir = "smart_ns_ctrl_fw_error_regs"
```

Comment:
The subdirectory name is human-unreadable (at least for me) - if I would see that name in a directory structure, I won't guess what it contains. Maybe if we want to store outputs of per block devices commands, name the directory `devices` or `block_devices`? See other usages of that:  https://github.com/sosreport/sos/search?q=%22subdir%3D%22

Reply:
Yeah, I'm personally caught up on the subdirectory structure.

I don't mind adding subdirs here in general, but the question to me is do we make these per-command subdirs or per-device? An analogous use case in sos currently is containers throughout several plugins, or perhaps moreso the different resources that OpenShift supports. The current theme has been to make subdirs based on resources (in this case, devices) rather than commands.